cfy_manager/main.py

```diff
@@ -650,12 +651,14 @@ def _get_components_list():
     # Can't easily use list comprehension here because this is a list of lists
     ordered_components = []
     for service in ordered_services:
-        ordered_components.extend(SERVICE_COMPONENTS[service])
+        for component in SERVICE_COMPONENTS[service]:
+            if not include_components or component in include_components:
```

Comment:
`if not include_components`
it can be extracted outside of loop

Reply:
i dont think it can, unless you mean something like `if not: ordered.extend(service) else: for ...`, which i think is uglier :P

Reply:
sorry, I looked at the prev commit

Reply:
Well, those are small lists, so prettier it is